packages/adapter/addon/rest.ts

```diff
@@ -1063,7 +1099,7 @@ class RESTAdapter extends Adapter.extend(BuildURLMixin) {
     @param {Object} options
     @return {Object}
   */
-  ajaxOptions(url, method, options) {
+  ajaxOptions(url: string, method: string, options: ConfidentDict<any>) {
```

Comment:
options is a Dict<unknown> right?

Reply:
Ya that seems right.  The non class `function ajaxOptions` is a ConfidentDict.